src/pybind/mgr/insights/tox.ini

```diff
@@ -8,6 +8,7 @@ minversion = 2.8.1
 deps =
     pytest
     mock
+    six>=1.14.0
```

Comment:
might want to note down why this change is not cherry-picked from master, i think it's likely due to that master has dropped the python2 support. see also 6bdb280954bda42cc8de37165a368cc30450f858

Reply:
ack